xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3590,15 +3590,17 @@ void CVideoPlayer::SetSpeed(float speed)
   if (!CanSeek())
     return;
   
-  m_newPlaySpeed = speed * DVD_PLAYSPEED_NORMAL;
+  int iSpeed = static_cast<int>(speed * DVD_PLAYSPEED_NORMAL); //! @todo
```

Comment:
DVD_PLAYSPEED_NORMAL is defiend as 1000. A speed of 0.9 resulted in new playspeed of 900. I want to keep this behaviour.